src/pages/edit_resource/access_and_licence_step/AccessFields.tsx

```diff
@@ -68,12 +102,14 @@ const AccessFields: FC<AccessFieldsProps> = ({ setAllChangesSaved }) => {
                   data-testid="access-dropdown-menu"
                   variant="filled"
                   select
+                  disabled={disabledUserInput}
```

Comment:
Ser for meg egentlig at hvis man har valgt at arbeidet er rettighetsklarert med intern lisens så burde lisens wizarden og lisensvelgeren også disables.

Reply:
verdt å spørre litt rundt om det er aktuelt. Men samtidig har vi vel lagt opp til å kunne overstyre wizarden